### ISSUE TITLE
Black screen playback

### ISSUE THREAD

**Issue by jonathanm-tkf** [bitmovin/bitmovin-player-react-native#163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Good morning, we are having a problem with the player, on certain occasions, this goes black screen, and we can not see the video, apparently, this happens in Android 10, 9, 8, .... in versions like 11, 12, and 13 this does not occur. Do you have any idea what could be happening? Any comments on how to fix it would be appreciated.

https://github.com/bitmovin/bitmovin-player-react-native/assets/67608970/504d6229-126c-48af-a727-c72d9b08e827

I have been looking at the native code and I see that they have a reference, I attach the code extract in case it has something to do with the problem, thank you very much.

### Expected Behavior

not see black screen and reproduce video normally

### Steps To Reproduce

1 - Run basic implementation
2 - Confirm you are on Android 10, 9, 8 .... or lower

### What platform(s) are you experiencing the issue on?

- [X] Android
- [ ] Android TV / Fire TV
- [ ] iOS
- [ ] tvOS

### Player React Native SDK version

0.7.2

### Device / Environment

- Device:
- Android 9

### Additional information / Code snippets / Screenshots

RNPlayerView.kt
```java
**
     * Try to measure and update this view layout as much as possible to
     * avoid layout problems related to React or old layout values present
     * in `playerView` due to being previously attached to a different parent.
     */
    override fun requestLayout() {
        super.requestLayout()
        post {
            measure(
                MeasureSpec.makeMeasureSpec(width, MeasureSpec.EXACTLY),
                MeasureSpec.makeMeasureSpec(height, MeasureSpec.EXACTLY)
            )
            layout(left, top, right, bottom)
        }
    }
```

**Comment by christophaigner** [comment]
Hi @jonathanm-tkf,

Thank you for reporting this, we appreciate it.
We were able to reproduce this issue occasionally as well and will look into it.

Chris

**Comment by jonathanm-tkf** [comment]
Hi @christophaigner if I can help in any way, please do not hesitate to let us know, unfortunately, it is something that is detrimental to our production output, we look forward to your support and thank you very much.

**Comment by christophaigner** [comment]
Hi @jonathanm-tkf,

Thank you for your patience. We've just released [v0.8.0](https://github.com/bitmovin/bitmovin-player-react-native/releases/tag/v0.8.0), which contains a fix/workaround for this problem (see #174 for more details).

Let us know if you still see the issue with the latest version.

Chris

**Comment by jonathanm-tkf** [comment]
Ni @christophaigner thanks for the support I will keep you informed of any developments.

**Comment by jonathanm-tkf** [comment]
Hi @christophaigner unfortunately it didn't fix the temporary fix, any other suggestions?

![image](https://github.com/bitmovin/bitmovin-player-react-native/assets/67608970/55f9ed07-14ec-4b98-b2fc-f67b5b82f661)

**Comment by jonathanm-tkf** [comment]
Samsung Galaxy S20 - Android 10

**Comment by christophaigner** [comment]
Hi @jonathanm-tkf,

Thank you for testing again with the latest version and reporting back.
This appears to be a different issue; in the [video](https://github.com/bitmovin/bitmovin-player-react-native/issues/163#issue-1807950829) you originally posted the Player wasn't visible at all - now the Player seems to be visible but the video isn't.

Could you please share more details about your environment & setup (eg is this running on a device farm, emulator, physical device, etc.) and if possible the stream as well? You can send the stream to christoph.aigner@bitmovin.com in case you don't want to post it here.
Also, please share any other details you believe could help us reproduce the issue.

Finally, can you reproduce this issue with our sample app as well?

Thank you again for your help!

Chris

**Comment by jonathanm-tkf** [comment]
Hello, @christophaigner thank you very much for your help.

There are several questions, I will answer

1 - It is a physical device or at least that's what Browserstack says, I don't have a device in my possession that has the same characteristics.

2 - Unfortunately they are client streams, which are not public and you need several steps to get to play them, maybe what I could do is simulate the same scenario in a controlled environment and with public urls, can you share me some urls repository of MPD streams?

Thank you very much for your help again

Translated with www.DeepL.com/Translator (free version)

**Comment by krocard** [comment]
Hi @jonathanm-tkf,

The issue reported in your last comment (https://github.com/bitmovin/bitmovin-player-react-native/issues/163#issuecomment-1658898640) is most probably different from the issue you originally reported. I highly recommend opening a new issue and continuing the discussion there. Here are a few thoughts:

From the screenshot it looks like this is an app playing a series ("*Episode 2*"). Most commercial movies and series are protected by [DRM](https://bitmovin.com/drm-meaning-explained/).
DRM forbid the content to be captured in any way. The content only appears on the physical screen of the device.
Could your screenshot come from a software screen recording of a real device (no reflection, no perspective), rather than a camera pointed toward the device? Which would explain why the content is back.

> can you share me some urls repository of MPD streams?

You can find reference DRM streams on our [DRM demo page](https://bitmovin.com/demos/drm).

**Comment by jonathanm-tkf** [comment]
Hi @krocard 
Currently, we are having DRM protected content previously implemented and we know what you are referring to in this issue, but this is not the case, this content I show does not have DRM enabled, the second not minor point is I understand it is another scenario, I will try to handle it in another ticket and I want to recreate the development inside your Example app to see if it is a problem of the implementation or respectively of the player.
This unfortunately takes some time because we need a couple of queries to external APIs.
Third and not least, unfortunately, we do not have that physical device which makes it impossible for us to use that method of reproduction for that we use BrowserStack, in the documentation of the latter explicitly says that they are not simulators, but are real devices, but perhaps you are right about the mirror.

Translated with www.DeepL.com/Translator (free version)

**Comment by christophaigner** [comment]
As discussed, I'm closing this issue as the original problem seems to have been fixed.
Please open a new issue with more details about the problem you've mentioned [here](https://github.com/bitmovin/bitmovin-player-react-native/issues/163#issuecomment-1658898640).

Thank you!

**Comment by jonathanm-tkf** [comment]
Hi @christophaigner  thank you very much again for the help but unfortunately, we are going to have to reopen the problem, because it is happening again without much effort to play it, I attach a video of more details as well as the open repository so you can check the source code, unfortunately, I don't feel confident to publish an app that definitely users can't see the video as soon as they enter the video.

https://github.com/bitmovin/bitmovin-player-react-native/assets/67608970/6c476b7e-bd21-4cbf-8b5f-56099efb963b

Steps to reproduce
1- Install APP
2- Open Landscape Fullscreen Britbox
3- See the error, doesn't have any buttons, the player not is created
4- Public repository - branch error
https://github.com/take0ffmedia/BBX-bitmovin-player-RN/tree/error
5- Drive APK release if you need https://drive.google.com/file/d/1K0qOGipOFSRNGnQOt_-LWoI3HRABOVcM/view?usp=sharing

Thank you very much for your help again and we will be in touch if you have any questions or comments.

**Comment by jonathanm-tkf** [comment]
Hi guys we have the same error on v.0.9.0

You can see the error on APK

https://drive.google.com/file/d/10dN9BeE2GebnioRbACxcIw-gra8hIaib/view?usp=sharing

**Comment by christophaigner** [comment]
Hi @jonathanm-tkf,

This should be fixed in [v0.9.1](https://github.com/bitmovin/bitmovin-player-react-native/releases/tag/v0.9.1).
Please let us know if you still see the issue.

Chris
